aws/resource_aws_emr_instance_group.go

```diff
@@ -116,13 +122,15 @@ func readEmrEBSConfig(d *schema.ResourceData) *emr.EbsConfiguration {
 	return result
 }
 
+// Please also see https://docs.aws.amazon.com/goto/WebAPI/elasticmapreduce-2009-03-31/AddInstanceGroups
```

Comment:
Could you put this instruction above line 137?
A create call *might* do multiple calls, like creating the instance group, attach options... it hardly depends on the API-side, but hey... 🤷‍♂️ 😄